### DISCUSSION TITLE
چطور میتونم فرمت تاریخ انتخاب شده رو تغییر بدم؟

### DISCUSSION THREAD

**Discussion by amir2002002** [majidh1/JalaliDatePicker#57] [Question] [answered]
با سلام و تشکر ... پس از انتخاب یک تاریخ توسط تقویم ، مقادیر با slash " / " از هم جدا شده اند . چطور میتوان بجای slash از dash " - " استفاده کرد؟

توضیحات در قسمت readme و ویژگی separatorChars کافی نبود و منجر به تغییر خروجی نشد.

**Comment by majidh1** [comment, marked as answer]
سلام خدمت شما از کانفیگ درستی استفاده کردید به شکل زیر مقدار دهی کنید کانفیگ رو

```js
separatorChars: {
        date:"-"
    }
```
